src/main/java/me/coley/j2h/loader/Export.java

```diff
@@ -0,0 +1,23 @@
+package me.coley.j2h.loader;
+
+import me.coley.j2h.modle.Configuration;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Marshaller;
+import java.io.IOException;
+
+public class Export {
+
+    // Current demo code for reference ready for when the UI has export button.
+    public static void main(String[] args) throws JAXBException, IOException {
```

Comment:
Demo & Testing code can be moved over to the [testing directory](https://github.com/Col-E/Java2HTML/blob/master/src/test/java/) in the form of JUnit tests.

Reply:
I have moved this into the test directory. But note it's not intended as a unit test. It would be bad practice if it was because writes to system out. It's only there for now to show the gist of what will need doing, but piped to file.